client/scss/components/_more_info_link.scss

```diff
@@ -1,11 +1,18 @@
 .more-info-link {
   @include font('HNM6');
-
   display: inline-flex;
   color: color('action-green-2');
   text-decoration: none;
   align-items: center;
   margin-top: $spacing-unit * 3;
+
+  .icon {
+    // The icon's viewBox is 0 0 12 13 so x=12 y=13
+    // Default icon height is (2 * $v-spacing-unit)
+    // TODO: find a way that we don't have to set
+    // both svg height and width for IE -- this is silly/brittle.
```

Comment:
Don't think there is a way unless we don't use inline svg, but put it in an img  tag instead